code/modules/admin/verbs/adminhelp.dm

```diff
@@ -251,7 +251,7 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 	ticket_interactions = list()
 	player_interactions = list()
 
-	addtimer(CALLBACK(src, .proc/add_to_ping_ss, 3 MINUTES)) //SKYRAT EDIT ADDITION - Ticket Ping
+	addtimer(CALLBACK(src, .proc/add_to_ping_ss, 2 MINUTES)) //SKYRAT EDIT ADDITION - Ticket Ping | this is not responsible for the notification itself, but only for adding the ticket to the list of those to notify.
```

Comment:
So why change it to 2 minutes here, exactly? Won't that cause the first ping to happen after two minutes? Or is it additional time before the first ping?

Reply:
This is exclusively the time when the ticket will be added to the system with a ping.
The pings themselves only happen when the ping subsystem fires